### DISCUSSION TITLE
Is it possible to extend / modify the CameraOrbitControl class?

### DISCUSSION THREAD

**Discussion by dougcharnley** [jnsmalm/pixi3d#191] [Q&A] [answered]
I'd like to be able to customise and have fine grain control over the CameraOrbitControl class so that I can do thing like make pinching zoom more slowly, make orbiting more or less sensitive to cursor movement and limit zoom levels. Is there an established way to do this? 

Many thanks 

Doug

**Comment by jnsmalm** [comment, marked as answer]
Hey!

Sorry for late reply. Right now the easiest thing is to just copy the source from https://github.com/jnsmalm/pixi3d/blob/develop/src/camera/camera-orbit-control.ts and add the features you need.

**Reply by dougcharnley** [reply]
OK fair enough - thanks for the response!
